src/plugins/intel_cpu/tests/functional/single_layer_tests/convolution_backprop_data.cpp

```diff
@@ -224,9 +224,14 @@ class DeconvolutionLayerCPUTest : public testing::WithParamInterface<DeconvLayer
 
         std::tie(kernel, stride, padBegin, padEnd, dilation, convOutChannels, padType, outPadding) = basicParamsSet;
 
-        if (additionalConfig[ov::hint::inference_precision.name()] == ov::element::bf16) {
+        if (configuration.count(ov::hint::inference_precision.name()) &&
+                configuration[ov::hint::inference_precision.name()].as<ov::element::Type>() == ov::element::bf16) {
             inType = outType = prec = ElementType::bf16;
             rel_threshold = 1e-2f;
+        } else if (configuration.count(ov::hint::inference_precision.name()) &&
+                configuration[ov::hint::inference_precision.name()].as<ov::element::Type>() == ov::element::f16) {
+            inType = outType = prec = ElementType::f16;
+            rel_threshold = 0.00001f;
```

Comment:
suggest to increase to `(1e-2)/8=0.00125` too

Reply:
will update it

Reply:
already fix